src/Core/Math/Ellipsoid.js

```diff
@@ -56,10 +56,10 @@ Ellipsoid.prototype.setSize = function setSize(size) {
 };
 
 
-Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto) {
+Ellipsoid.prototype.cartographicToCartesian = function cartographicToCartesian(coordCarto, normal) {
     // var n;
     var k = new THREE.Vector3();
-    var n = this.geodeticSurfaceNormalCartographic(coordCarto);
+    const n = normal ? normal.clone() : this.geodeticSurfaceNormalCartographic(coordCarto);
```

Comment:
You could replace this with:
```js
const n = normal ? normal.clone() : coordCarto.geodesicNormal.clone();
```
Or even better, you could avoid changing the API and you'll get the same result by doing:
```js
const n = coordCarto.geodesicNormal.clone();
```